src/main/java/org/haffson/adventofcode/utils/CheckStringisEmpty.java

```diff
@@ -0,0 +1,20 @@
+package org.haffson.adventofcode.utils;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CheckStringisEmpty {
+
+    private CheckStringisEmpty() {
+
+    }
+
+    public static String requireNonNullAndNonEmpty(String string) {
```

Comment:
Die Hilfsmethode finde ich erstmal gut. Man muss in so einem Fall allerdings

- gucken, was es ist libraries schon gibt 
- tests dafür schreiben
- dokumentieren